common/source/docs/common-non-gps-navigation-landing-page.rst

```diff
@@ -16,6 +16,7 @@ These are the available options that allow a vehicle to estimate its position wi
 
     Intel RealSense T265 <common-vio-tracking-camera>
     MarvelMind Beacons <common-marvelmind>
+    Nooploop Beacons <common-nooploop>
```

Comment:
Remove this...instead place it in the common-master-features.rst file as an upcoming feature since its master branch only at this point